qsimcirq_tests/qsimcirq_test.py

```diff
@@ -248,6 +291,22 @@ def test_iterable_qubit_order():
   )
 
 
+@pytest.mark.parametrize('mode', ['noiseless', 'noisy'])
+def test_preserve_qubits(mode: str):
```

Comment:
nit: use a bool
```suggestion
@pytest.mark.parametrize('add_noise', [False, True])
def test_preserve_qubits(add_noise: bool):
```
then `if add_noise:`

Reply:
Using a string here is intentional. It ensures that the test shows up as `test_preserve_qubits[noisy]` in pytest logs, instead of the more cryptic `test_preserve_qubits[True]`.